src/components/routes/Routers.jsx

```diff
@@ -0,0 +1,21 @@
+import { Route, Routes } from "react-router-dom";
+import { Home, Notes } from "../../pages/";
+import { Trash } from "../Trash/Trash";
+import { Archives } from "../Archives/Archives";
+import { Labels } from "../Labels/Labels";
+import { Note } from "../Notes/Note";
+

```

Comment:
you already import   this  component into your index.js file , so export this component into index.js file 
and  import this from index.js file

Reply:
yes , will do it in the next PR.